src/links/links.service.ts

```diff
@@ -27,19 +31,52 @@ export class LinksService {
     }
   }
 
-  async findOrginalUrl(shortUrl: Link['shortUrl']): Promise<string | null> {
+  async findOrginalUrl(
+    shortUrlCode: Link['shortUrlCode'],
+  ): Promise<string | null> {
     try {
       const link = await this.prisma.link.findUnique({
-        where: { shortUrl },
+        where: { shortUrlCode },
       });
       return link.originalUrl;
```

Comment:
**suggestion**: Consider returning an empty array instead of `null` when no links are found for a user to avoid potential null reference errors.